pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -47,6 +48,7 @@
 , odbcPackages ? [ unixODBC ]
 , opensslPackage ? openssl
 , wxPackages ? [ libGL libGLU wxGTK xorg.libX11 wrapGAppsHook3 ]
+, patchShell ? true
```

```suggestion
, patchShell ? true
```
Sorry, I did not mean this.
I don't think the user needs to tweak this.
I meant just to check for versions directly.

---

I think the only users of those options are the sibling `24.nix`, `25.nix` files?

I can't see a good way to check the versions directly, other than using the major-version nix files themselves.

i.e. `25.nix` specifies `version = "25.3.2.12";`.